### PR TITLE
Update xrefs to docs site for new docs URL

### PR DESCRIPTION
**Pull request by Loquacity**
Complete list in issue: https://github.com/SUSE/spacewalk/issues/9711

### CODE REVIEW

File: modules/retail/pages/retail-migration-slepostosuma.adoc [hunk 1]
```diff
@@ -3,8 +3,9 @@
 
 
 This section describes migrating from an existing {slepos} 11 installation to a new {susemgr} installation.
-You can perform this migration all at once by creating a data dump in a single file, and then moving it to the new server; see <<retail_migr.sect.slepostosuma.s1.complete>>.
-Alternatively, you can perform the migration in stages by creating a data dump for each branch, and moving them to the new server one by one; see <<retail_migr.sect.slepostosuma.s1.bbb>>.
+You can perform this migration all at once by creating a data dump in a single file, and then moving it to the new server.
+
+Alternatively, you can perform the migration in stages by creating a data dump for each branch, and moving them to the new server one by one..
```
Comment (keichwa) [inline modules/retail/pages/retail-migration-slepostosuma.adoc]:
```suggestion
Alternatively, you can perform the migration in stages by creating a data dump for each branch, and moving them to the new server one by one.
```
Oohh, one dot too much!

File: modules/retail/pages/retail-next.adoc [hunk 2]
```diff
@@ -9,9 +9,9 @@ If you need further information or support, try one of these options.
 [[retail.sect.next.docs]]
 == More Documentation
 
-For {susemgr} documentation, visit https://www.suse.com/documentation/suse-manager-4/
+For {susemgr} documentation, visit https://documentation.suse.com/suma/4.0/
```
Comment (keichwa) [inline modules/retail/pages/retail-next.adoc]:
```suggestion
For {susemgr} documentation, visit https://documentation.suse.com/suma/4.0/.
```
it belongs here

File: modules/retail/pages/retail-next.adoc [hunk 3]
```diff
@@ -9,9 +9,9 @@ If you need further information or support, try one of these options.
...
 
-For legacy {slepos} or {smr} documentation, see https://www.suse.com/documentation/slepos11/index.html or https://www.suse.com/documentation/suse-manager-for-retail.
+For legacy {slepos} or {smr} documentation, see https://documentation.suse.com/sle-pos/11-SP3/ or https://documentation.suse.com/suma-retail/3.2/.
 Note, however, that {smr} documentation supersedes this information.
```
Comment (keichwa) [inline modules/retail/pages/retail-next.adoc]: Drive-by finding: this is nicely unclear.

File: modules/retail/pages/retail-migration-slepostosuma.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/retail/pages/retail-migration-slepostosuma.adoc]: Well, that's a change ;)

File: modules/retail/pages/retail-next.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/retail/pages/retail-next.adoc]: :laughing:

File: modules/retail/pages/retail-next.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/retail/pages/retail-next.adoc]: Yeah, I was trying to avoid too much editing. I'll quickly split this into two, though.

**Comment by Loquacity** [comment]
> `link:http://` (w/o space inbetween) is special adoc markup. With space it would be wrong or soemthing different. We'd better remove it then.

Oh, yeah, I was moving too fast, clearly.

> 
> The rest is even more cosmetic.
> 
> Thanks for sorting out the mess!

No problem. Sadly not as easy as a copy/paste, I had to go and locate every section in every book ...

**Comment by Loquacity** [comment]
Another thought: we should specifically add updating these links to our maintenance plan. The docs site requires version numbers to make the URLs work. I've updated them all to 15 SP1, but that will have to be reviewed regularly.

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> No problem. Sadly not as easy as a copy/paste, I had to go and locate every section in every book ...

Yes, teadious work!  I hope "they" will not change it in the near future
again...  Ok, there are these version numbers as you already pointed out...

-- 
Karl Eichwalder            SUSE Software Solutions Germany GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 247165, AG München)
